web/src/app/registration/registration.component.scss

```diff
@@ -0,0 +1,12 @@
+$defaultFont: 'Arial', 'Helvetica', 'sans-serif';
+
+form, #formContainer {
```

Comment:
Might be better to apply the flex to only one of these, since it is a bit confusing how these two different elements affect each other.

Reply:
Removed "form" and left only "#formContainer".